common/abstracts/_variables.scss

```diff
@@ -26,6 +26,15 @@ $blue-magenta: #000064;
 $orange: hsla(39, 100%, 50%, 1);
 $orange-overlay: hsla(39, 100%, 50%, 0.25);
 
+$topicImageBlueMagenta: rgba(0, 0, 100, 0.45);
+$topicImageBlueMagentaHover: rgba(0, 0, 100, 0.65);
+$topicImageBurgundy: rgba(153, 0, 43, 0.45);
+$topicImageBurgundyHover: rgba(153, 0, 43, 0.66);
+$topicImageOrange: rgba(255, 166, 0, 0.45);
+$topicImageOrangeHover: rgba(255, 166, 0, 0.65);
+$topicImageWhite: rgba(255, 255, 255, 0.45);
+$topicImageWhiteHover: rgba(255, 255, 255, 0.65);
```

Comment:
Would use `1` as the last number for the `stroke`, and then `0.25` for the `fill`. And also perhaps change the naming form `topicImageOrangeHover` to `topicImageOrangeFill` for instance. In the editor an `topicImageOrangeHover` with a different opacity (higher than 0.25) could be used for `:hover` - adding a comment about this further down as well